kubetest/e2e.go

```diff
@@ -762,18 +762,18 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, logexporterGCSPath, wg *sync.WaitGroup, provider, dump string) {
```

Comment:
You also need to change the code that calls this codepath, in like ~250. Please use o.logexporterGCSPath for as a value for the new parameter.